include/aspect/simulator_signals.h

```diff
@@ -263,8 +257,7 @@ namespace aspect
      * allows modification of the assembly objects active in this simulation.
      */
     boost::signals2::signal<void (const SimulatorAccess<dim> &,
-                                  aspect::internal::Assembly::AssemblerLists<dim> &,
-                                  std::vector<std_cxx11::shared_ptr<internal::Assembly::Assemblers::AssemblerBase<dim> > > &)>
+                                  aspect::Assemblers::Manager<dim> &)>
```

Comment:
I guess we can not avoid this breaking change. Do we care?

Reply:
No. We are in for a major release. Moreover: Anybody who used this interface used it to connect a function to a signal that now no longer exists, so they have to change their implementation anyway, we might as well fix the interface.